### PR TITLE
Register

### PR DESCRIPTION
**Pull request by Samet91**

### CODE REVIEW

File: src/app/Pages/Dashboard.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default function Dashboard(): JSX.Element {
     if (response.status === 200) {
       console.log('added in database')
     } else {
-      console.log('scheiße gelaufen')
```
Comment (F-Kirchhoff) [inline src/app/Pages/Dashboard.tsx:41]: hahaha :D

File: src/app/Pages/Register.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ username, password }),
+    })
+    if (response.status === 201) {
+      navigate('/login')
+    } else {
+      alert('Benutzer vorhanden! Bitte einen neuen Benutzernamen auswählen')
+    }
```
Comment (F-Kirchhoff) [inline src/app/Pages/Register.tsx]: nice!

File: src/app/components/UserForm/UserFormStories.tsx [hunk 3]
```diff
@@ -9,6 +9,15 @@ export default {
 function handleLoginSubmit(username: string, password: string): void {
   console.log(`Logged in ${username}, ${password}`)
 }
-export const UserLogin = (): JSX.Element => (
+
+function handleRegisterSubmit(username: string, password: string): void {
+  console.log(`Logged in ${username}, ${password}`)
```
Comment (F-Kirchhoff) [inline src/app/components/UserForm/UserFormStories.tsx:14]: maybe dont show the pw in clear text ?

File: src/server.ts [hunk 4]
```diff
@@ -16,13 +16,13 @@ app.use(express.json())
 app.post('/api/costs', async (req, res) => {
   const newCosts = req.body
   await getCollection().insertOne(newCosts)
-  res.status(200).send('Newcost was added')
+  res.status(200).send('Newcost were added')
```
Comment (F-Kirchhoff) [inline src/server.ts:19]: "New costs" or "New cost'

**Review by F-Kirchhoff — APPROVED** [review]
Wow! Great stuff

File: src/server.ts (hunk 4 above)
Comment (Samet91) [reply src/server.ts:19]: thank you for your code review.. yes it´s better New cost, because it is always only one

**Pull request merged**